ethereum/PFP.sol

```diff
@@ -64,16 +64,16 @@ library MerkleProof {
 contract PFP is ERC721A, Ownable, VRFConsumerBase {
     using Strings for uint256;
     
-    uint256 public MAX_TOKENS = 20000;
-    string public baseExtension = ".json";
+    uint256 public constant MAX_TOKENS = 20000;
+    string public constant BASE_EXTENSION = ".json";
```

Comment:
MAX_TOKENS and BASE_EXTENSION can be set to constants as they are set in the definition and do not change in the contract. This will save 2,100 gas on the read of each.